packages/integration-test-app/tests/InteropCSModuleTests.tsx

```diff
@@ -30,7 +30,8 @@ functionTest('StringConstant', () => {
   assert.equal(msg, 'Hello World');
 });
 
-functionTest('PointConstant', () => {
+// #7808: Broken on Hermes Release
+functionTest.skip('PointConstant', () => {
   const point = testModuleCs.PointConstant;
```

Comment:
FYI @dannyvv 

Haven't looked closely at this. Any idea why these tests might be failing on Hermes/Release?